src/mlmd/LineageTypes.ts

```diff
@@ -4,13 +4,16 @@
 import {Artifact, Execution} from "..";
 
 export type LineageCardType = 'artifact' | 'execution';
-export interface LineageRow {
```

Comment:
How are these different?

Reply:
Not really different, in an intermediate state, I updated this type to use type expressions, but in the end. It was fine to keep it as interface.

I'll restore unneeded change here.